README.md

```diff
@@ -52,26 +52,33 @@ The following figure shows the typical CoatRack architecture, the CoatRack web a
 
 ![CoatRack architecture overview](./spring-boot/admin/src/main/resources/static/images/coatrack-architecture-overview.png)
 
-## Install
+## Installation for local usage
 
-Scripts for building and running CoatRack are available in the root folder of the code base. To build CoatRack, use:
+Go to the docker-compose-setup folder and execute:
 
-```console
-./build.sh
-```
-To run the CoatRack web application, use:
+ ```sh
+ .init.sh
+ ```
 
-```console
-./run-admin-application.sh
+The script only needs to be executed once at the very first time to initialize the docker volumes. After that, executing 'docker-compose up/down' is sufficient. After starting up, the CoatRack web application will be accessible at `http://localhost:8080`.
```

Comment:
If it is only required once, I would separate into two commands: 
```
./init.sh
docker-compose up
```
I would put that 'docker-compose up/down' also as commands to execute, e.g.
"[...]
After the initialisation, use the following command to start CoatRack:
```
docker-compose up
```
"

Reply:
Done as suggested.